### ISSUE TITLE
Setting a header image via the customizer fails

### ISSUE THREAD

**Issue by nerrad** [deliciousbrains/wp-amazon-s3-and-cloudfront#263]
Recently we started seeing this happen:

1. Open customizer, select header pane, click to add image.
2. Upload image, success!
3. Next step is to select and apply crop.  Get the usual fail response from WP.

Here's a screenshot of the error (not useful but provides context)

https://www.evernote.com/l/AAMMyKMUEGdKSrqKNfajtd6U75yV_VIkcNMB/image.png

In the network tab of developer tools (chrome) there is what looks like a dump of the http stream (sample):

```
����JFIF``��;CREATOR: gd-jpeg v1.0 (using IJG JPEG v80), quality = 90
��C




��C		

����"��	
���}!1AQa"q2���#B��R��$3br�	
%&'()*456789:CDEFGHIJSTUVWXYZcdefghijstuvwxyz���������������������������������������������������������������������������	
���w!1AQaq"2�B����	#3R�br�
$4�%�&'()*56789:CDEFGHIJSTUVWXYZcdefghijstuvwxyz��������������������������������������������������������������������������?�V�Hc3�8/� ��$�<&Bv�8��ԑ�
	S�?<�w��:�B�e!NG��|Q䷡\�0Vg6�����h`�'��<7�Q��f��bp
```

at the end of the stream is this error message from WP (some parts obfuscated):

```
<b>Warning</b>:  fopen(https://d12m9erqbesehq.cloudfront.net/wp-content/uploads/sites/3/2015/08/07130922/cropped-IMG_1788.jpg): failed to open stream: HTTP wrapper does not support writeable connections in <b>{path to wp}/class-wp-image-editor.php</b> on line <b>424</b><br />
{"success":false,"data":{"message":"Image could not be processed. Please go back and try again."}}
```

We are running the plugin in a WordPress multisite environment. 

Let me know what else you need to troubleshoot.

**Comment by nerrad** [comment]
Oh, and want to add, that cropping, editing images via the normal media routes works fine.  It's just the customizer header where we are having problems.

**Comment by polevaultweb** [comment]
@nerrad could you post the diagnostic log from the support tab please, so I can try to recreate

**Comment by polevaultweb** [comment]
@nerrad no worries, managed to recreate :) Working on a fix...

**Comment by nerrad** [comment]
hey @polevaultweb would you be able to point me to the commit that resolved this?

**Comment by nerrad** [comment]
or is this something that is fixed in the PRO version?

**Comment by polevaultweb** [comment]
hi @nerrad. the version of Offload S3 0.9.3 that has just been released was a tiny release for the Pro version, we have been working on 0.9.4 which has the fix for your issue and is coming soon.

**Comment by A5hleyRich** [comment]
Resolved in version 0.9.4.
